src/components/Layout/footer/footer.js

```diff
@@ -99,23 +127,32 @@ export function Footer() {
               weekly.
             </p>
           </div>
-          <form className="mt-6 sm:flex sm:max-w-md lg:mt-0">
+          <form
+            onSubmit={handleSubmitEmail}
+            className="mt-6 sm:flex sm:max-w-md lg:mt-0 "
+          >
             <label htmlFor="email-address" className="sr-only">
               Email address
             </label>
-            <input
-              type="email"
-              name="email-address"
-              id="email-address"
-              autoComplete="email"
-              required
-              className="w-full min-w-0 appearance-none rounded-md border-0 bg-white/5 px-3 py-1.5 text-base text-white shadow-sm ring-1 ring-inset ring-white/10 placeholder:text-gray-500 focus:ring-2 focus:ring-inset focus:ring-indigo-500 sm:w-56 sm:text-sm sm:leading-6"
-              placeholder="Enter your email"
-            />
-            <div className="mt-4 sm:mt-0 sm:ml-4 sm:flex-shrink-0">
+            <div className=" min-w-0 inline-flex justify-start ring-1 ring-inset ring-white/10 border-0 bg-white/5 rounded-md focus-within:ring-2 focus-within:ring-indigo-500">
```

Comment:
subscribe button and input are in the same line? Will it be changed it? or keep it as the original version?

Reply:
make it as we discussed it today for legaliser